### ISSUE TITLE
Locations of tools

### ISSUE THREAD

**Issue by yashpalgoyal1304** [ShareX/ShareX#5335]
#### Summary

The knowledge of these ways will be extremely useful to edit the images from other programs directly into sharex's tools. For the moment, i am only concerned about the image annotation tool, and image editor, but yeah, the scope of expansion is still there.
- What is/are the way(s) to invoke the tools via address/external program and have them send data (image) directly to these tools?
For example, the image editor tool when opened from within sharex shows this dialog. Is there a way that this can be triggered/invoked directly via some other program. 
![image](https://user-images.githubusercontent.com/19423063/104868751-4b51f580-596a-11eb-8a60-57813c8b5271.png)
- Basically how can sharex tools be "actions" for some other tool? 
- And also, how can they be invoked via Windows Open With popup and its Look for another app dialog?

---
My guesses:
1. .exe package
Seems highly unlikely that each tool is packaged as its own .exe, so no, this doesnt seem the way to go
2. executable name
Maybe u have specified different executables name doing different things - like done by https://exiftool.org/#running - but it also seems unlikely
2. CLI
Maybe the tools have some way to be used via cli - tried searching https://getsharex.com/docs/command-line-arguments but found nothing related.

**Comment by yashpalgoyal1304** [comment]
The way to go can also be to internally provide a selection for which tool to use with the input ( in #5332 ) - that is after the data (image, text etc) is "opened with" sharex

**Comment by yashpalgoyal1304** [comment]
Ahhh, I remember now why I muted this repo. Bot shitting around all the time.

Sent from Mail for Windows 10
